src/app/market/sell/seller-listing/seller-listing.component.ts

```diff
@@ -0,0 +1,80 @@
+import { Component, Input, Output, EventEmitter } from '@angular/core';
+import { MatDialog } from '@angular/material';
+import { Router } from '@angular/router';
+import { Log } from 'ng2-logger';
+
+import { RpcStateService } from 'app/core/rpc/rpc-state/rpc-state.service';
+import { ModalsService } from 'app/modals/modals.service';
+import { TemplateService } from 'app/core/market/api/template/template.service';
+
+import { DeleteListingComponent } from '../../../modals/delete-listing/delete-listing.component';
+
+import { Template } from 'app/core/market/api/template/template.model';
+import { Listing } from 'app/core/market/api/listing/listing.model';
+import { Status } from '../status.class';
+
+@Component({
+  selector: 'app-seller-listing',
+  templateUrl: './seller-listing.component.html',
+  styleUrls: ['./seller-listing.component.scss']
+})
+export class SellerListingComponent {
+
+  @Output() callLoad: EventEmitter<any> = new EventEmitter();
+
+  public status: Status = new Status();
+  log: any = Log.create('seller-listing.component');
+
+  @Input() listing: Listing;
+
+  constructor(
+    public dialog: MatDialog,
+    private router: Router,
+    private rpcState: RpcStateService,
+    private modals: ModalsService,
+    private template: TemplateService
+  ) {}
+
+  getStatus(status: string) {
+    return [this.status.get(status)];
+  }
+
+  confirmDeleteListing(template: Template): void {
+    const dialogRef = this.dialog.open(DeleteListingComponent);
+    dialogRef.componentInstance.templateToRemove = template;
+    dialogRef.afterClosed().subscribe(
+      () => this.callLoad.emit()
```

Comment:
Can you perhaps get element reference to the SellerListingComponent element and delete it?
That way we don't have to reload the page.